go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/Microsoft/ethr
+
+require (
+	github.com/mattn/go-runewidth v0.0.3
+	github.com/nsf/termbox-go v0.0.0-20181027232701-60ab7e3d12ed
+	golang.org/x/sys v0.0.0-20181206074257-70b957f3b65e
```

Comment:
Any particular reason why this is added here?

Reply:
we use this package somewhere in plt_darwin.go file, I made sure we have exact same revision as Gopkg.lock

Reply:
OK, thanks for the clarification!